src/main/java/org/eln2/mc/client/gui/PlotterScreen.kt

```diff
@@ -121,31 +133,37 @@ class PlotterScreen(private val cells : ArrayList<CellInfo>) : Screen(TextCompon
                 RenderSystem.setShaderColor(1.0f, 1.0f, 1.0f, 1.0f)
             }
 
-            blit(pPoseStack, it.pos.x, it.pos.y, 0f, 0f, 32, 32, 32, 32)
+            drawNanoseconds += measureNanoTime {
+                blit(pPoseStack, it.pos.x, it.pos.y, 0f, 0f, texSize, texSize, texSize, texSize)
+            }
 
-            val xMin = (it.pos.x + posX) * scale
-            val yMin = (it.pos.y + posY) * scale
-            val xMax = scale * texSize.toFloat() + xMin
-            val yMax = scale * texSize.toFloat() + yMin
+            drawList.add(PoseStackExtensions.Union4i(it.pos.x, it.pos.y, texSize, texSize))
+
+            // check if the mouse is hovering over the cell
             if(pMouseY > yMin && pMouseY < yMax && pMouseX > xMin && pMouseX < xMax) {
                 toolTipCell = it
-                return@forEach
             }
         }
 
+        /*
+        if(drawList.isNotEmpty()){
+            drawNanoseconds = measureNanoTime {
+                pPoseStack.blitMultiple(drawList)
+            }
+        }*/
+
```

Comment:
What's this for?

Reply:
this was used by the optimized drawing method. It can be removed.